docs/src/explanations/naming.md

```diff
@@ -153,35 +171,8 @@ Users who desire a prefix are encouraged to provide one as [described below](#pr
 
 ### Prefixing
 
-As shown above, the compiler plugin automatically attempts to prefix some of your signals for you. However, you as a
-user can also add your own prefixes. This is especially useful for ECO-type fixes where you need to add some logic to a module
-but don't want to influence other names in the module.
-
-In the following example, we prefix additional logic with "ECO", where `Example4` is pre-ECO and `Example5` is post-ECO:
-
-```scala mdoc
-class Example4 extends Module {
-  val in = IO(Input(UInt(2.W)))
-  val out = IO(Output(UInt()))
-
-  val add = in + in + in
-
-  out := add + 1.U
-}
-
-class Example5 extends Module {
-  val in = IO(Input(UInt(2.W)))
-  val out = IO(Output(UInt()))
-
-  val add = in + in + in
-
-  out := prefix("ECO") { add + 1.U + in }
-}
-```
-```scala mdoc:verilog
-emitSystemVerilog(new Example4)
-emitSystemVerilog(new Example5)
-```
```

Comment:
I deleted the ECO example only because I literally cannot figure out a reliable way to make temporaries show up in the Verilog (other than naming the signals myself but that sort of defeats the purpose).

I do think we should have an ECO example but we need to make sure it captures enough complexity to actually be interesting. I'm also worried that firtool's much more aggressive optimizations will make this sort of ECO stuff harder to do so it may just not really be showable at the moment (without adding some sort of optimization barrier primitive).

If anyone can come up with a good example, I'm happy to add it back.